components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -226,72 +231,43 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
-                                    <a className="gp-link" href="https://www.gitpod.io/pricing#cost-estimator">
-                                        Estimate costs
-                                    </a>
-                                </span>
-                            </div>
-                        </div>
-                        <div className="flex justify-end mt-6 space-x-2">
-                            {stripePortalUrl && (
-                                <a href={stripePortalUrl}>
-                                    <button className="secondary" disabled={!stripePortalUrl}>
-                                        View Past Invoices ↗
-                                    </button>
-                                </a>
-                            )}
-                            <button onClick={() => setShowBillingSetupModal(true)}>Upgrade Plan</button>
-                        </div>
-                    </div>
-                )}
-                {showUpgradeUser && (
+                {(showUpgradeTeam || showUpgradeUser) && (
                     <div className="flex flex-col mt-4 p-4 rounded-xl bg-gray-50 dark:bg-gray-800">
-                        <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Current Plan</div>
-                        <div className="mt-1 text-xl font-semibold flex-grow text-gray-600 dark:text-gray-400">
-                            {usageLimit > 500 ? "Open Source" : "Free"}
-                        </div>
-                        <div className="mt-4 flex space-x-1 text-gray-400 dark:text-gray-500">
-                            <Check className="m-0.5 w-5 h-5 text-orange-500" />
-                            <div className="flex flex-col">
-                                <span className="font-bold text-gray-500 dark:text-gray-400">{usageLimit} credits</span>
-                                <span>
-                                    {usageLimit / 10} hours of Standard workspace usage.{" "}
-                                    <a className="gp-link" href="https://www.gitpod.io/pricing#cost-estimator">
-                                        Estimate costs
-                                    </a>
-                                </span>
-                            </div>
-                        </div>
+                        {usageLimit > 0 && (
+                            <>
+                                <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Current Plan</div>
+                                <div className="mt-1 text-xl font-semibold flex-grow text-gray-600 dark:text-gray-400">
+                                    {usageLimit > 500 ? "Open Source" : "Free"}
+                                </div>
+                                <div className="mt-4 flex space-x-1 text-gray-400 dark:text-gray-500">
+                                    <Check className="m-0.5 w-5 h-5 text-orange-500" />
```

Comment:
**suggestion**: This was probably changed in a past PR, but could we increase the size of the checkmark here?

```suggestion
                                    <Check className="m-0.5 w-8 h-5 text-orange-500" />
```

| BEFORE | AFTER |
|-|-|
| <img width="477" alt="checkmark-before" src="https://user-images.githubusercontent.com/120486/213674719-95a6d763-35a2-4d8c-9994-64e86c729bff.png"> | <img width="477" alt="checkmark-after" src="https://user-images.githubusercontent.com/120486/213674726-1c9c5e5c-2028-4d7a-8cce-ef85b4190ccd.png"> |